numbat/modules/units/misc.nbt

```diff
@@ -115,6 +115,11 @@ unit swimmingpool: Volume = 50 m × 25 m × 2 m # Olympic-size swimming pool (FR
 @aliases(rackunits, RU: short, U: short)
 unit rackunit: Length = 0.04445 meter
 
+@metric_prefixes
+@aliases(darcys, darcies, D: short)
+@url("https://en.wikipedia.org/wiki/Darcy_(unit)")
+unit darcy: Length^2 = (1 bar / 1 atmosphere) × micrometer²
```

Comment:
I was wondering if we should introduce `dimension Permeability = Length^2`, but (1) there is also *electric* permeability with a different physical dimension, which might be confusing. And (2) this new dimension would show up as an alternative whenever the result is an area — which might also be confusing. So let's leave it at `Length^2` for now.